templates/rhpam74-kieserver-externaldb.yaml

```diff
@@ -676,6 +687,8 @@ objects:
             containerPort: 8888
             protocol: TCP
           env:
+          - name: WORKBENCH_SERVICE_NAME
```

Comment:
What should happen when user doesn't set this environment variable, considering that this variable is not mandatory and doesn't have default value?

Reply:
@spolti Correct me if I'm wrong.
@sutaakar  If this environment variable is undefined, the hook script will ignore the 'curl' command. In such case, this KIE Server will be treated as unmanaged.

Reply:
correct, the WORKBENCH_CONTROLLER and KIE_SERVER_ID must exists for the hook be triggered.
there there is only the KIE_SERVER_ID the configmap will be updated only, the controller event (curl request) will not be fired.